multiworld/envs/mujoco/sawyer_xyz/sawyer_door.py

```diff
@@ -150,15 +150,17 @@ def compute_rewards(self, actions, obs):
             raise NotImplementedError("Invalid/no reward type.")
         return r
 
-    def reset(self):
-        if self.reset_counter % self.num_resets_before_door_and_hand_reset == 0:
+    def reset_model(self):
+        if not self.reset_free:
             self._reset_hand()
             self._set_door_pos(0)
         goal = self.sample_goal()
         self.set_goal(goal)
-        self.reset_counter += 1
         self.reset_mocap_welds()
         return self._get_obs()
+    
+    def reset(self):
```

Comment:
Why is this necessary? The subclass implements this (see `mujoco_env.MujocoEnv`)?

Reply:
@mdalal2020 Bump